Sources/Rendering/Core/Prop/index.js

```diff
@@ -17,6 +17,7 @@ function vtkProp(publicAPI, model) {
       Math.max(val.getMTime(), acc)
     , 0));
 
+  publicAPI.getProps = () => publicAPI;
```

Comment:
There must be a typo here?

Reply:
By default a prop is a prop right? So getProps should by default return itself. Or are you saying it should be [publicAPI] ?

Reply:
You are right, it is correct. I was expecting another method call and thought you forgot the method.